examples/step-6/step-6.cc

```diff
@@ -191,8 +191,8 @@ Step6<dim>::Step6 ()
 // To be fair, such exceptions about still used objects are not particularly
 // popular among programmers using deal.II, since they only tell us that
 // something is wrong, namely that some other object is still using the object
-// that is presently being destructed, but most of the time not who this user
-// is. It is therefore often rather time-consuming to find out where the
+// that is presently being destroyed, but most of the time not who this using
```

Comment:
Yes, indeed -- that sounded strange but I couldn't come up with something better in the moment. That's why we need native English speakers here :-)

I fixed the place. Thanks for the suggestion.